src/parameterizations/vertical/MOM_opacity.F90

```diff
@@ -431,24 +443,13 @@ function SW_pen_frac_morel(chl_data)
   ! appropriate when using an interactive ecosystem model that predicts
   ! three-dimensional chl-a values.
   real :: Chl, Chl2         ! The log10 of chl_data in mg m-3, and Chl^2 [nondim]
-  real, dimension(6), parameter :: &
-    V1_coef = (/0.321,  0.008, 0.132,  0.038, -0.017, -0.007/) ! Penetrating fraction coefficients [nondim]
 
   Chl = log10(min(max(chl_data,0.02),60.0)) ; Chl2 = Chl*Chl
-  SW_pen_frac_morel = 1.0 - ( (V1_coef(1) + V1_coef(2)*Chl) + Chl2 * &
-       ((V1_coef(3) + Chl*V1_coef(4)) + Chl2*(V1_coef(5) + Chl*V1_coef(6))) )
+  SW_pen_frac_morel = 1.0 - ( (CS%SW_pen_frac_coef(1) + CS%SW_pen_frac_coef(2)*Chl) + Chl2 * &
+                              ((CS%SW_pen_frac_coef(3) + Chl*CS%SW_pen_frac_coef(4)) + &
+                               Chl2*(CS%SW_pen_frac_coef(5) + Chl*CS%SW_pen_frac_coef(6))) )
 end function SW_pen_frac_morel
 
-!>   This sets the blue-wavelength opacity according to the scheme proposed by
-!! Manizza, M. et al, 2005.
-function opacity_manizza(chl_data)
-  real, intent(in)  :: chl_data !< The chlorophyll-A concentration [mg m-3]
-  real :: opacity_manizza !< The returned opacity [m-1]
-!   This sets the blue-wavelength opacity according to the scheme proposed by Manizza, M. et al, 2005.
-
-  opacity_manizza = 0.0232 + 0.074*chl_data**0.674
```

Comment:
Where did this function go?

Reply:
this looks like an outdated function, also possibly incorrect since the Manizza 2005 scheme is multiband. The opacity is set on line 388 (new code) using a multiband scheme

Reply:
`Opacity_manizza()` was unused, outdated and using hard-coded dimensional constants, so I eliminated it.  I would be OK with restoring it with a new name, like `opacity_manizza_archaic()` as a part of a managed obsoleting process, but I do not see what purpose it serves.